README.md

```diff
@@ -0,0 +1,17 @@
+# Akka Platform Operator Charts
+
+This repository contains Helm charts for the [Akka Platform](https://developer.lightbend.com/docs/akka-platform-guide/).
```

Comment:
I think you can link to `[Akka Platform Operator](https://developer.lightbend.com/docs/akka-platform-guide/deployment/)`, where we will have an overview page.

Reply:
done